content/docs/composition-vs-inheritance.md

```diff
@@ -78,15 +80,15 @@ function App() {
 }
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/gwZOJp?editors=0010)
+[**Incercați in CodePen**](https://codepen.io/gaearon/pen/gwZOJp?editors=0010)
 
-React elements like `<Contacts />` and `<Chat />` are just objects, so you can pass them as props like any other data. This approach may remind you of "slots" in other libraries but there are no limitations on what you can pass as props in React.
+Elementele React precum `<Contacts />` și `<Chat />` sunt doar obiecte, astfel le poți trece ca si proprietati, ca orice alte date. Această abordare vă poate aminti de "slots" din alte librari, dar nu există nicio limitare a ceea ce puteți trece ca elemente de recuzită în React.
```

Comment:
I would not translate props here as elemente de recuzita. I would leave it like props.